include/picongpu/particles/particleToGrid/derivedAttributes/MomentumDensity.def

```diff
@@ -22,11 +22,13 @@
 #include "picongpu/simulation_defines.hpp"
 
 #include "picongpu/particles/traits/SpeciesEligibleForSolver.hpp"
+#include "picongpu/plugins/misc/ComponentNames.hpp"
 #include "picongpu/traits/SIBaseUnits.hpp"
 
 #include <pmacc/static_assert.hpp>
 #include <pmacc/traits/HasIdentifiers.hpp>
 
+#include <array>
```

Comment:
```suggestion
```
This include should not be required by the new code. Is this a fix for a co-located inaccuracy?

Reply:
fixed